### ISSUE TITLE
[BUG] Movie export only saves about 8 seconds (99f) of video / audio

### ISSUE THREAD

**Issue by Jose-Moreno** [pencil2d/pencil#549]
### --Issue Summary--

When exporting a movie in either mp4 or avi format, with or without sound, regardless of timeline duration and FPS count the movie will get cut at the 8 seconds mark.
### --Actual Results

After exporting a timeline with two frames, the first in frame 1 and the second in frame 1000, both mp4 and avi, I always got 99frames of footage in my final video.
### --Expected Results--

The exported movie should be as long as the last frame in the timeline dictates. so if there's a frame in the 1000th frame, it should export 1000 frames instead of 99.
### --Steps to reproduce--
1. create new file
2. Select the bitmap layer.
3. Draw a stroke in frame 1.
4. Create a second keyframe in frame number 300.
5. Draw another stroke.
6. Import sound optional.
7. Go to File > Export > Movie. Select either MP4 or AVI, and press "save"
8. It will take some time to export, but you'll be able to notice by the filesize.
9. Play the exported video and notice the 8 second duration.
### --System Information--
- Pencil2D Version:
  Nightly Build Oct 10 2016
- Operating System:
  Windows 7 Ultimate 64bits
- RAM Size:
  4 GB
- CPU:
  AMD Phenom II x4 965 BE

**Comment by chchwy** [comment]
should be fixed, need to verify in next nightly build

**Comment by Jose-Moreno** [comment]
@chchwy Alright had some free time and tested this. It works in Nov 2nd 2016 Nightly Build. I exported an MP4 that had about 1500+ frames and everything was exported properly.

There's a problem I'm beginning to see and it's the way Pencil2D is handling image memory. After importing all the animated GIF's I could, I ended up using around 2 gigs of memory. In fact importing the images consumed more memory than anything else. We might have to consider looking into ways to optimize how image cache is handled, but i'll leave that for another issue. I'll close this now.
